src/constants/selectOptions.ts

```diff
@@ -0,0 +1,32 @@
+export const CATEGORY_WITH_ENTIRE: CategoryWithEntire[] = [
+  "전체",
+  "한식",
+  "중식",
+  "일식",
+  "아시안",
+  "양식",
+  "기타",
+];
```

Comment:
이 부분은 `CATEGORY`를 이용해서 아래처럼 적어볼 수 있을거 같아요
```suggestion
export const CATEGORY_WITH_ENTIRE = ["전체", ...CATEGORY];
```